dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/ServiceConfig.java

```diff
@@ -724,4 +736,45 @@ public DubboBootstrap getBootstrap() {
     public void setBootstrap(DubboBootstrap bootstrap) {
         this.bootstrap = bootstrap;
     }
+
+    protected void delayExport() {
+        doExport();
+    }
+
+    private void duplicateExportCheck() {
```

Comment:
I think it would be better to extract the service bean triple check to another class, such as DubboBootstrap / ConfigManager.
The `serviceBeanId` field name is not appropriate enough, it may be misunderstood as spring bean id/name.

It is recommended to use the non-static Set<String> field to save the state instead of using a static object. Static objects will bring a lot of state problems, don't use it unless necessary

Reply:
when I checked the test cases, I also found that writing this way will cause a lot of issues.
thank you for your guidance